### DISCUSSION TITLE
How can I get in touch with Myntra executive?

### DISCUSSION THREAD

**Discussion by Monika1995ug** [ROCm/ROCm#3472] [General]
How can I get in touch with Myntra executive? 8IO6I_99I94 Hello everyone,is it a solution to manage ldap servers integrated to a fortigate, but with an account in protected users group.It seems that once the account i use to connect my fortigate and my AD servers is put in the ..

**Comment by Monika1995ug** [comment]
How can I get in touch with Myntra executive?8106199194

**Comment by darren-amd** [comment]
Hi @Monika1995ug,

Could you clarify what your question is? Seems to be incomplete.
